FastSurferCNN/run_prediction.py

```diff
@@ -104,22 +110,22 @@ def args2cfg(args: argparse.Namespace) -> Tuple[yacs.config.CfgNode, yacs.config
 
 
 def removesuffix(string: str, suffix: str) -> str:
```

Comment:
this function can be removed, and in places where it is called, replace `removesuffix(string, suffix)` with `string.removesuffix(suffix)`

Reply:
I guess this is a standalone function, this function is not being called anywhere in run_prediction.py.